main.go

```diff
@@ -44,16 +43,11 @@ func main() {
 	// Initialized developer mode flag for the logger
 	log.InitializeLogger(config.IsLogJSON(), config.GetLogLevel())
 
-	keycloakConfig := keycloak.Config{
```

Comment:
Remove config options from Configuration and update DC and CM example if no longer used.

Reply:
Good point https://github.com/fabric8-services/fabric8-notification/pull/73/commits/45693c25e3295d6006f57f7de3a139e27043afb3